activestorage/app/models/active_storage/variant_with_record.rb

```diff
@@ -8,6 +8,7 @@ class ActiveStorage::VariantWithRecord
   attr_reader :blob, :variation
   delegate :service, to: :blob
   delegate :content_type, to: :variation
+  delegate :forced_disposition_for_serving, :content_type_for_serving, to: :image
```

Comment:
Likewise, what are your thoughts on including `ActiveStorage::Blob::Servable` here too?  `VariantWithRecord` already implements `filename` and delegates `content_type`.

With that approach, the first argument to `send_blob_stream` will always be `is_a?(ActiveStorage::Blob::Servable)`, which is kind of nice from a typing perspective.

Reply:
Yes I agree as well!